src/App.vue

```diff
@@ -140,12 +143,11 @@ export default defineComponent({
     },
     uiThemeCategory: {
       get() {
-        let zclProps = this.$store.state.zap.selectedZapConfig?.zclProperties
-        // Picking the first category in the case of multi-protocol(zigbee/matter)
-        if (Array.isArray(zclProps) && zclProps.length > 0) {
-          return zclProps[0].category
+        if (this.$store.state.zap.isMultiConfig) {
+          return 'multiprotocol'
         } else {
-          return this.$store.state.zap.selectedZapConfig?.zclProperties.category
+          return this.$store.state.zap.selectedZapConfig?.zclProperties[0]
```

Comment:
Are you sure `zclProperties` is an array when it is not multi-protocol? I remember some variables being array or not depending on multi-protocol or single protocol.